packages/frontend/src/routes/SignWrapper.js

```diff
@@ -0,0 +1,104 @@
+import BN from 'bn.js';
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import SignTransferRetry from '../components/sign/SignTransferRetry';
+import SignTransactionDetailsWrapper from '../components/sign/v2/SignTransactionDetailsWrapper';
+import SignTransactionSummaryWrapper from '../components/sign/v2/SignTransactionSummaryWrapper';
+import { Mixpanel } from '../mixpanel';
+import {
+    addQueryParams,
+    handleSignTransactions,
+    selectSignFeesGasLimitIncludingGasChanges,
+    SIGN_STATUS,
+    selectSignStatus,
+    selectSignCallbackUrl,
+    selectSignMeta,
+    selectSignTransactionHashes
+} from '../redux/slices/sign';
+
+export function SignWrapper() {
+    const dispatch = useDispatch();
+
+    const [showTransactionDetails, setShowTransactionDetails] = useState(false);
+    const [insufficientNetworkFee, setInsufficientNetworkFee] = useState(false);
+
+    const signFeesGasLimitIncludingGasChanges = useSelector(selectSignFeesGasLimitIncludingGasChanges);
+    const signStatus = useSelector(selectSignStatus);
+    const signCallbackUrl = useSelector(selectSignCallbackUrl);
+    const signMeta = useSelector(selectSignMeta);
+    const transactionHashes = useSelector(selectSignTransactionHashes);
```

Comment:
<3 I like the consistency of var naming to selector names.  Could you make a post in the dev channel about this and call this out to everyone else so they can start following this pattern also?